bazel/cas/client_test.go

```diff
@@ -0,0 +1,54 @@
+package cas
+
+// To generate required mockgen files for these tests, from the Top Level github.com/twitter/scoot dir:
+//
+//	(prerequisite: go get google.golang.org/genproto/googleapis/bytestream)
+// 	mockgen google.golang.org/genproto/googleapis/bytestream ByteStreamClient,ByteStream_ReadClient > bazel/cas/mock_bytestream/bsclient_mock.go
+//
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"golang.org/x/net/context"
+	"google.golang.org/genproto/googleapis/bytestream"
+	remoteexecution "google.golang.org/genproto/googleapis/devtools/remoteexecution/v1test"
+
+	"github.com/twitter/scoot/bazel"
+	"github.com/twitter/scoot/bazel/cas/mock_bytestream"
+	"github.com/twitter/scoot/snapshot/store"
+)
+
+func TestClientRead(t *testing.T) {
+	f := &store.FakeStore{}
```

Comment:
It's not immediately clear what this is used for.

Reply:
this was useless; removed